Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -92,6 +92,17 @@ public override object DeserializeObject(object value, Type type)
                 {
                     if (ReflectionUtils.GetTypeInfo(type).IsEnum)
                     {
+                        //first try to get all custom attributes
```

Comment:
Would be good for @shiftkey or @Haacked to comment on whether they're happy with this change to deserializer.  Is this the most elegant way of doing this and would there be any concerns around performance?  Just wondering, since it's looping for every value of every enum and checking for custom attribute, when the majority of enums/values dont have one.